Resources/Prototypes/Entities/Clothing/Ears/headsets.yml

```diff
@@ -118,6 +130,8 @@
     - Medical
   - type: Sprite
     sprite: Clothing/Ears/Headsets/medical.rsi
+  - type: Clothing
+    sprite: Clothing/Ears/Headsets/medical.rsi
```

Comment:
these shouldnt actually be necessary

Reply:
But it does overwrite the `sprite` of the parent and use their own sprite.